module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -548,10 +564,12 @@ public function getElements()
     /**
      * Return form recipient(s).
      *
+     * @param array $postParams Posted form data
```

Comment:
Are you adding a parameter but not using it?

Reply:
No, not here. But we'll extend `getRecipient` to use `$postParams` in our code base.

Reply:
Thanks, that makes sense! I've added a `@SuppressWarnings(PHPMD.UnusedFormalParameter)` annotation so this won't cause warnings in PHPMD.